src/Gravatar.php

```diff
@@ -86,8 +152,85 @@ private function createEmailHash(string $email): string
         return md5(strtolower(trim($email)));
     }
 
+    private function buildOptions(array $options): array
+    {
+        // Image size
+        if (array_key_exists('s', $options)) {
```

Comment:
I went with the code duplication for the short / long option names rather than try to smash them down into one option name or the other. Future work might be to see how Gravatar behaves if you pass both long and short option name with different values and adjust in here as appropriate.